uai2017/python/populate_papers_data.py

```diff
@@ -0,0 +1,163 @@
+    return notes
+
+
+def get_submitted_paper_details(submitted_paper_notes):
+    """
+    Get details of all the papers submitted in a list
+    :param submitted_paper_notes:
+    :return:
+    """
+    paper_details_list=[]
+    paper_details_list.append(["Paper ID","Paper Title","Abstract","Author Names","Author Emails","Subject Areas","Conflict Reasons",
+                               "Files","Supplementary File","Related Submissions","AC suggestions","Toronto Paper Matching System Agreement"])
+    for note in submitted_paper_notes:
+        paper_id = note.id
+        paper_title = note.content["title"]
+        paper_abstract = note.content["abstract"]
+        author_names= ";".join(note.content["authors"])
+        author_emails = ";".join(note.content["authorids"])
+        subject_areas = note.content["TL;DR"]
+        conflict_reasons = ";".join(note.content["conflicts"])
+        files = note.content["pdf"]
+        supplementary_files=""
+        related_submission=""
+        ac_suggestions=""
+        toronto_paper_matching_system_agreement=""
+        paper_details_list.append([paper_id,paper_title,paper_abstract,author_names,author_emails,
+                                            subject_areas,conflict_reasons,files,supplementary_files,related_submission,
+                                            ac_suggestions,toronto_paper_matching_system_agreement])
+    return  paper_details_list
+
+
+def get_paper_reviewer_score_tree(submitted_paper_notes,dict_reviewers_email):
+    """
+    Get an XML tree for paper reviewer score
+    :param submitted_paper_notes:
+    :param dict_reviewers_email:
+    :return:
+    """
+    root=etree.Element("reviewermatching")
+    meta_data_notes=get_paper_metadata_notes(get_paper_names(submitted_paper_notes))
+    for index in range(len(submitted_paper_notes)):
+        paper_id = submitted_paper_notes[index].id
+        paper_element = etree.Element("submission",submissionId=str(paper_id))
+        reviewers_score = {x["reviewer"]:x["score"] for x in meta_data_notes[index].content["reviewers"]}
+        for reviewer,data in dict_reviewers_email.iteritems():
+            score = 0
+            if reviewer in reviewers_score:
+                score=reviewers_score[reviewer]
+            paper_element.append(etree.Element("metareviewer",email = data[3],score=str(score)))
+        root.append(paper_element)
+    return etree.ElementTree(root)
+
+
+def get_all_member_ids():
+    members = filter(lambda id: id != "~Super_User1" and id != "~",
+                     map(lambda x: x.id, openreview.get_groups(regex="~.*")))
+    return  members
+
+
+
+def get_all_members_data():
+    """
+    Get a dictionary of each member with its details
+    :param members:
+    :return:
+    """
+    members = get_all_member_ids()
+    dict_reviewer_data = {}
+    for member in members:
+        member_note = openreview.get_note(id=member)
+        member_first_name= member_note.content["first"]
```

Comment:
@agankur21 here is an example of a profile data: https://openreview.net/user/profile?id=~Melisa_TestBok1

you need to follow the new schema to get the user profile data